app/models/user.rb

```diff
@@ -0,0 +1,12 @@
+class User < ApplicationRecord
+  VALID_EMAIL_REGEX = /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i.freeze
+
+  validates :name, presence: true,
+    length: {maximum: Settings.validations.name_255,minimum: Settings.validations.name_6}
```

Comment:
```suggestion
    length: {maximum: Settings.validations.name_255, minimum: Settings.validations.name_6}
```

đổi các giá trị `name_255` -> `length.digist_225` ri nhé, để tái sử dụng

Reply:
Dạ, để e sửa liền ạ hehe